nextest-runner/src/reporter.rs

```diff
@@ -686,10 +686,15 @@ impl<'a> TestReporterImpl<'a> {
             TestEvent::TestRetry {
                 test_instance,
                 run_status,
+                delay_before_next_attempt,
             } => {
                 if self.status_level >= StatusLevel::Retry {
-                    let retry_string =
-                        format!("{}/{} RETRY", run_status.attempt, run_status.total_attempts);
+                    let retry_string = format!(
+                        "{}/{} RETRY in {}",
+                        run_status.attempt,
+                        run_status.total_attempts,
+                        humantime::Duration::from(*delay_before_next_attempt)
+                    );
```

Comment:
So the issue here is that there's a limit of 12 characters here, and something like "1/3 RETRY in 1s" exceeds that limit.

I've been thinking about this and I think the way I'd probably want to display this is, if there is a delay:

```
TRY 1/3 FAIL [<duration>]  <binary name> <test name>
       DELAY [ in 1.000s]  <binary name> <test name>
```

And then, when the next attempt is started:

```
   2/3 RETRY               <binary name> <test name>
```

If there is no delay, then, the `DELAY` line is skipped. for consistency:

```
TRY 1/3 FAIL [<duration>]  <binary name> <test name>
   2/3 RETRY               <binary name> <test name>
```

For its implementation, this would also need `TestEvent` to be extended, and specifically `TestRetry` to be split into two separate events:

* an attempt just failed, will retry in the future with maybe a delay
* the next attempt has started.